src/quicr_client_raw_session.cpp

```diff
@@ -534,6 +534,25 @@ QuicRClientRawSession::publishNamedObjectFragment(
   throw std::runtime_error("UnImplemented");
 }
 
+void
+QuicRClientRawSession::fetchNamedObject(const quicr::Namespace &context, const quicr::Name &name) {
+
+  std::lock_guard<std::mutex> lock(session_mutex);
+
+  auto transaction_id = messages::create_transaction_id();
+
+  if (!subscribe_state.count(context)) {
+      throw std::runtime_error("fetchNamedObject:Lookup Context not found");
+  }
+
+  // encode fetch
+  messages::MessageBuffer msg{};
+  messages::Fetch fetch{ transaction_id, context, name } ;
+  msg << fetch;
+  const auto& state = subscribe_state.at(context);
+  transport->enqueue(state.transport_context_id, state.transport_stream_id, msg.take());
+}
+
```

Comment:
Alternatively, only passing `Namespace` is still a viable way to retrieve a specific name as well, given the sig bits of namespace are 128, it would be equivalent.

Reply:
so fetch is for a full name and the api should point to that .. so what would be the recommendation

Reply:
My recommendation would be to only pass `Name`, since a `namespace_map` can find an entry given a `Name`.